File: Orbi-App/Assets/Scripts/Game/Server/Model/Inventory.cs
using System;
using System.Collections.Generic;
using UnityEngine;

namespace ServerModel
{
    [Serializable]
    public class Inventory: AbstractModel
    {
        public List<InventoryItem> items = new List<InventoryItem>();
        public List<GameObjectTypeCategory> categories = new List<GameObjectTypeCategory>();

    }

    
}

File: Orbi-Server/src/main/java/foundation/softwaredesign/orbi/model/Inventory.java
package foundation.softwaredesign.orbi.model;

import javax.xml.bind.annotation.XmlRootElement;
import java.util.ArrayList;
import java.util.List;

/**
 * @author Lucas Reeh <lr86gm@gmail.com>
 */
@XmlRootElement
public class Inventory {

    private List<InventoryItem> items = new ArrayList<>();
    private List<GameObjectTypeCategory> categories = new ArrayList<>();

    public List<InventoryItem> getItems() {
        return items;
    }

    public void setItems(List<InventoryItem> items) {
        this.items = items;
    }

    public List<GameObjectTypeCategory> getCategories() {
        return categories;
    }

    public void setCategories(List<GameObjectTypeCategory> categories) {
        this.categories = categories;
    }
}
